### PR TITLE
made it more clear for windows users

### PR DESCRIPTION
**Pull request by iheartkode**
Docs for windows is not very clear.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +144,13 @@ recommended to install the most stable build.
 Be sure to declare a environment variable named `HOME` pointing to your user
 directory `C:\Users\<username>`. Then you can clone Spacemacs in this directory.
 
+You can use the following git command to clone it into the .emacs.d folder, if you already have one make sure to delete it.
+then run.
+```
+git clone --recursive http://github.com/syl20bnr/spacemacs ~/.emacs.d
+```
```
Comment (robbyoconnor) [inline README.md]: You do not need `--recursive` anymore. So it is actually 
`git clone http://github.com/syl20bnr/spacemacs ~/.emacs.d

**Comment by robbyoconnor** [comment]
in addition to the above, this is against `master` -- next time, please do all pull requests against `develop`. Please read the [contribution guidelines](https://github.com/syl20bnr/spacemacs/blob/develop/CONTRIBUTING.md) which was linked when you opened this pull request ;)

**Comment by robbyoconnor** [comment]
This entire PR is pointless. Users are told to clone the repo at the top. Why do they need to be told twice? I like to assume users can retain information. Since this is against `master` and completely useless -- I suggest closing it.  I don't think instructions are not clear. It's not like it's a different command for windows. 

In the future do not do PRs against `master` -- We expect you to read the contribution guidelines prior to contributing ;)

**Comment by iheartkode** [comment]
:+1:

**Comment by robbyoconnor** [comment]
:heart:

**Pull request closed without merging**